tests/miniconda_install_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../scripts/newinstall.sh
+. "${DIR}/../scripts/newinstall.sh"
+
+# "stub" out cmd to prevent curl from being run
+# shellcheck disable=SC2034
+cmd="echo"
```

Comment:
I realise you're not adding `cmd` to `newinstall.sh` here, just taking advantage of it. However, it did cause me a moment of confusion before I worked out that it is _supposed_ to be undefined in normal operation. Worth a comment?